addons/web_editor/static/tests/field_html_tests.js

```diff
@@ -290,7 +290,15 @@ QUnit.module('web_editor', {}, function () {
             });
 
             await testUtils.form.clickEdit(form);
+            await new Promise((resolve)=>setTimeout(resolve, 50));
             var $field = form.$('.oe_form_field[name="body"]');
+            const toolbar = document.querySelector('#toolbar');
+            async function click() {
+                // Selection's bounding client rectangle has crazy values during Qunit tests,
+                // which would make the editor hide the toolbar. Force show it before clicking.
```

Comment:
Wait, so we need this change because, with your changes based on `getBoundingClientRect`, the tests wouldn't work properly ? I'm quite perplex because there are a lot of other uses of `getBoundingClientRect` in Odoo and I don't think they encounter the same issue in tests.

Reply:
Sounds good, thanks ! 👍